wire-ios-sync-engine/Source/E2EI/CRL/CertificateRevocationListsChecker.swift

```diff
@@ -112,6 +109,7 @@ public class CertificateRevocationListsChecker: CertificateRevocationListsChecki
 
     private func checkCertificateRevocationLists(from distributionPoints: Set<URL>) async {
 
+        var checkE2EICertificateForSelfClient = false
```

Comment:
maybe this name makes more sense
```suggestion
        var shouldNotifyAboutRevokedCertificate = false
```